backend/src/transactions.js

```diff
@@ -52,7 +52,7 @@ async function getTransactionsList(limit, paginationIndexer) {
     paginationIndexer
   );
   if (transactionsList.length === 0) {
-    return undefined;
+    return [];
```

Comment:
We have to return an empty array to allow our ListHandler to work properly

Reply:
It might have been a comment in the code, actually